schema/migrations/control_60_delete_webhook_index.cql

```diff
@@ -0,0 +1,6 @@
+USE @@KEYSPACE@@;
+
+-- Delete "webhooks_by_token" index since "webhook_keys" table 
+-- serves that purpose
+
+DROP TABLE webhooks_by_token;
```

Comment:
Should this be "DROP INDEX" or is that not a thing?  Are we on [1.2](http://www.datastax.com/documentation/cql/3.0/cql/cql_reference/drop_index_r.html)?